### ISSUE TITLE
New order and recommended tags for OpenHistoricaMap

### ISSUE THREAD

**Issue by ooosssay** [bryceco/GoMap#798]
Possibly have `start_date=`, `end_date=`, and `sources=` be placed on top of description in the Common Tags page when the user has selected the OpenHistoricalMap server.

**Comment by 1ec5** [comment]
Specifically, in an OHM editing context, every feature whatsoever would have these fields somewhere near the top.

In the long run, OpenHistoricalMap/issues#777 would allow Go Map!! to avoid special-casing specific servers and fields.

**Comment by ooosssay** [comment]
Oh yeah that would definitely be better

**Comment by ooosssay** [comment]
Should I leave this open but change the title to something along the lines of “Future support for OHM id tagging schema”?

**Comment by bryceco** [comment]
If you're editing OHM you can use the "Customize Presets" button to add those missing fields to Common Tags.

**Comment by ooosssay** [comment]
Omg I somehow missed that button I didn’t even know that button existed until now

**Comment by bryceco** [comment]
Yeah, as the number of presets has grown that button has gotten harder and harder to notice.
